docs/src/prototypes/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  createComponent,
+  Button,
+  Text,
+  Tooltip,
+  ShorthandValue,
+  ButtonProps,
+  TextProps,
+} from '@stardust-ui/react'
+import * as copyToClipboard from 'copy-to-clipboard'
+import * as _ from 'lodash'
+import * as React from 'react'
+
+import Notification from './Notification'
+
+export type CopyToClipboardProps = {
+  attached?: boolean
+  pointing?: boolean
+  timeout?: number
+  value: string
+
+  button?: ShorthandValue<ButtonProps>
```

Comment:
should be `trigger` instead (trigger?: ReactElemenet), like in `Portal`. Need to clone existing element, using React.cloneElement...